.github/workflows/goreleaser-check.yml

```diff
@@ -1,4 +1,4 @@
-name: GoReleaser
+name: GoReleaser check
```

Comment:
Technically, this is unrelated, but it's been annoying me for several releases now.

Reply:
Helpful!!! 🙌